### ISSUE TITLE
Extracting data using OpenEO

### ISSUE THREAD

**Issue by shabarinath4** [WorldCereal/worldcereal-classification#229]
Data extracted worked using batch jobs

1. A small buffer was created around each lat lon to generate a polygon feature
2. The spatial extent was defined slightly larger than the polygon feature
3. 'dekad' frequency and 'max' composite was used

Issues

1. When larger spatial extent is defined (extent which covers multiple polygons), the job fails (Connection disconnected error). Currently each polygon is being given as a separate job. Is this ideal?
2. The number of concurrent jobs is 2. Can this be increased?

@kvantricht

**Comment by shabarinath4** [comment]
@kvantricht yes transitioning to the WorldCereal scripts will be good. I can setup the two scenarios for the call. Does tomorrow or Monday 1PM work for you?

In the current per polygon approach, I think I should be able to get data to start retesting

**Comment by kvantricht** [comment]
Tomorrow 1PM should work.

**Comment by kvantricht** [comment]
Data extraction pipeline now works. Evaluation going on. Some distribution shift still seen between Google Earth Engine and openEO.

**Comment by shabarinath4** [comment]
Issue seems to be the processing level of data. Using L1C distribution looks as expected, next redo model evaluation with OpenEO dataset
